doc/source/whatsnew/v0.19.0.txt

```diff
@@ -433,6 +433,15 @@ Deprecations
 - ``as_recarray`` has been deprecated in ``pd.read_csv()`` and will be removed in a future version (:issue:`13373`)
 - top-level ``pd.ordered_merge()`` has been renamed to ``pd.merge_ordered()`` and the original name will be removed in a future version (:issue:`13358`)
 
+
+.. _whatsnew_0190.prior_deprecations:
+
+Removal of prior version deprecations/changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- ``pd.to_timedelta`` has dropped the ``coerce`` parameter (:issue:`13602`)
```

Comment:
Done.  Old build to cancel <a href="https://travis-ci.org/pydata/pandas/builds/143620635">here</a>.